capstone_project/src/main/webapp/index.html

```diff
@@ -9,7 +9,7 @@
     <div id="content">
       <h1>STEP Pod 155 Capstone</h1>
       <p>This is the capstone project for STEP Pod 155.</p>
-      <p>Click <a href="pages/mapsTemplate.jsp">here</a> to explore our maps feature.</p>
+      <p>Click <a href="pages/maps.html">here</a> to explore our maps feature.</p>
```

Comment:
it looks like this is wrong? GitHub says the file path is mapsTemplate.html not maps.html

Reply:
@Leoism it gets changed to maps.html at run/deploy time. When the app is run/deployed, an interactive copy of mapsTemplate.html(called maps.html) is created and this is where the actual API key value is substituted in for 'CAPSTONE_API_KEY'. I did not push maps.html to GitHub since it contains the actual key.